src/components/table/README.md

```diff
@@ -901,6 +901,51 @@ event will not be emitted when the input, select, textarea is clicked (unless th
 (even when disabled)
 
 
+## Row select
```

Comment:
This title should be "Row select supoort"

Reply:
done